Library/Homebrew/test/dev-cmd/ruby_spec.rb

```diff
@@ -13,6 +13,13 @@
       .and not_to_output.to_stderr
   end
 
+  it "prints the version number", :integration_test do
```

Comment:
Note: it's almost never worth adding new integration tests for stuff like this because they are very slow.

Reply:
Noted.